packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -26,74 +27,112 @@ const drawerContent = ({ active_index, setActiveTabIndex }) => {
     );
 };
 
+const StatTile = props => (
```

Comment:
Nit, could we give full names, i.e. `StatisticsTile` or `StatisticTile`?

Reply:
sure.